r-package/grf/_pkgdown.yml

```diff
@@ -0,0 +1,76 @@
+url: https://github.com/grf-labs/grf
+navbar:
+  type: default
+  right:
+  - icon: fa-github fa-lg
+    href: https://github.com/grf-labs/grf
+
+## Function reference
+# Different forest types
+reference:
+  - title: Causal forest
+    contents:
+      - causal_forest
+      - tune_causal_forest
+      - tune_ll_causal_forest
+      - predict.causal_forest
+
+  - title: Instrumental forest
+    contents:
+      - instrumental_forest
+      - predict.instrumental_forest
+
+  - title: Regression forest
+    contents:
+      - regression_forest
+      - tune_regression_forest
+      - predict.regression_forest
+
+  - title: Local linear regression forest
+    contents:
+      - ll_regression_forest
+      - tune_ll_regression_forest
+      - predict.ll_regression_forest
+
+  - title: Boosted regression forest
+    contents:
+      - boosted_regression_forest
+      - predict.boosted_regression_forest
+
+  - title: Quantile forest
+    contents:
+      - quantile_forest
+      - predict.quantile_forest
+
+  - title: Custom forest
```

Comment:
Maybe we can avoid linking to custom forests here, I don't think it's useful from an end-users' perspective?